azerothcore/mod-ah-bot
# Selling of items with zero value as buy/sell price

## Changes Proposed:
- Introduced an heuristic that allows the bot to sell values with zero sell/buy value by evaluating their base price using the item level. The algorithm is not perfect but provides a good start for setting up a price to previously unsold items.

## Issues Addressed:

- Addresses no issues

## SOURCE:


## Tests Performed:

- Compile without errors
- Tested that items that were no previously sold are now present into the market at a price which is more or less aligned with common and uncommon items.


## How to Test the Changes:


1. Compile & install
2. Use the whitelist option as the following: `AuctionHouseBot.SellerWhiteList="2672, 769, 3173, 2673, 5503, 2674, 2675, 1015 3685, 3731, 3712, 12184, 12205, 12207, 35562, 12208, 20424, 2447, 765, 2450, 2453, 785, 3820, 3357, 3356, 3821, 3355, 8838, 3358, 8831, 8836, 8839, 13464, 13465, 22785, 22789, 2770, 2771, 2775, 2772, 2776, 3858, 7911, 10620, 2589, 2592, 4306, 4338, 14047, 6338, 11128, 11144, 16206, 10940, 11083, 11137, 11176, 16204, 10998, 11082, 11134, 11135, 11174, 11175, 22445, 2318, 783, 2319, 4232, 4234, 4235, 4304, 8169, 8170, 8171"`
3. Enter and see that enchanting reagents like `Vision dust` is not available to be sold at an acceptable price.
